src/component/Home.js

```diff
@@ -26,3 +26,8 @@ export default () => {
         </div>
     );
 };
+
+Home.propTypes = {};
```

Comment:
If there aren't any propTypes, you can just leave this off. Personal preference though.

Reply:
I was thinking that, but if there are no prop types does this flag if one is added? I've never thought about it before.

Reply:
I'm not sure what you mean. If you pass extraneous props that are not defined, they are just passed through. PropTypes does not check all props, it only checks for defined props and ignores the rest. If there are no `.propTypes`, it does no checking.